packages/trees/frontend/sirius-components-trees/src/treeitems/context-menu/TreeItemContextMenu.tsx

```diff
@@ -64,7 +65,18 @@ export const TreeItemContextMenu = ({
 
   const { selectionTargets } = useSelectionTargets();
 
-  const { loading, contextMenuEntries } = useContextMenuEntries(editingContextId, treeId, item.id);
+  const { getContextMenuEntries, loading, contextMenuEntries } = useContextMenuEntries();
+
+  useEffect(() => {
+    getContextMenuEntries({
+      variables: {
+        editingContextId,
+        representationId: treeId,
+        treeItemId: item.id,
+      },
+    });
+  }, []);
```

Comment:
As for diagrams, the query to get the context menu entries now has to be lazy to be able to trigger it only if the user entered a valid key combination.